packages/dart_frog_cli/lib/src/commands/dev/dev.dart

```diff
@@ -176,16 +183,28 @@ class DevCommand extends DartFrogCommand {
         final message = utf8.decode(_).trim();
         if (message.isEmpty) return;
 
-        /// Do not kill the process if the error is a warning from the SDK.
+        final isDartVMServiceAlreadyInUseError =
+            _dartVmServiceAlreadyInUseErrorRegex.hasMatch(message);
         final isSDKWarning = _warningRegex.hasMatch(message);
 
-        if (isSDKWarning) {
+        if (isDartVMServiceAlreadyInUseError) {
+          logger.err(
+            '$message '
+            '''Please, try specifying a different port using the `--dart-vm-service-port` argument when running `dart_frog dev`.''',
```

Comment:
Should we remove or keep the "Please" ?

So from:
```sh
Please, try specifying a different port using the `--dart-vm-service-port` argument when running `dart_frog dev`.
```

To:
```sh
Try specifying a different port using the `--dart-vm-service-port` argument when running `dart_frog dev`.
```

Any thoughts?

(cc: @scarletteliza)

Reply:
We can remove unless you feel strongly otherwise!